### ISSUE TITLE
Configuration warning unrecognized property 'ssmToEnvironment'

### ISSUE THREAD

**Issue by ats-org** [gozup/serverless-ssm-fetch#12]
Greetings,

I don't think this is a big problem cause it looks like the environment variables are getting propagated. however, there is a warning when running with sls offline

```
Serverless: Configuration warning:
Serverless:   at 'functions.handler': unrecognized property 'ssmToEnvironment'
Serverless:  
Serverless: Learn more about configuration validation here: http://slss.io/configuration-validation
```

```
functions:
  create:
      handler: src/handler
      events:
        - http:
            path: create
            method: post
            cors: true
      ssmToEnvironment:
        - DB_NAME
        - DB_USER
        - DB_PASSWORD
        - DB_HOST
        - DB_PORT
        - CORS_DOMAIN
```

**Comment by ats-org** [comment]
One thing that does happen is builds fail when adding:
configValidationMode: error

to the serverless.yaml

```
 Deprecation warning: Starting with next major, Serverless will throw on configuration errors by default. Adapt to this behavior now by adding "configValidationMode: error" to service configuration
            More Info: https://www.serverless.com/framework/docs/deprecations/#CONFIG_VALIDATION_MODE_DEFAULT
```

**Comment by ats-org** [comment]
It looks like the property just needs to be added to the constructor
https://www.serverless.com/framework/docs/providers/aws/guide/plugins/#extending-validation-schema

**Comment by invaderb** [comment]
Here is the PR to fix that issue
https://github.com/gozup/serverless-ssm-fetch/pull/13

**Comment by jestjosh** [comment]
Is this going to get merged?

**Comment by invaderb** [comment]
I just tagged @gozup in the PR for an update.

**Comment by gozup** [comment]
Hi @invaderb, @joshmello 

PR #13 has been merged and deployed on NPM under version 1.1.0

See ya!
